guides/common/modules/proc_migrating-to-external-databases.adoc

```diff
@@ -4,32 +4,31 @@
 Back up and transfer existing data, then use the `{foreman-installer}` command to configure {Project} to connect to an external PostgreSQL database server.
 
 .Prerequisite
-
-* You have installed and configured a PostgreSQL server on a {RHEL} server.
+* You have installed and configured a PostgreSQL server on a {EL} server.
 
 .Procedure
-
 . On {ProjectServer}, stop {Project} services:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # {foreman-maintain} service stop
 ----
-+
 . Start the *PostgreSQL* services:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # systemctl start postgresql
 ----
-+
 . Back up the internal databases:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# {foreman-maintain} backup online --skip-pulp-content --preserve-directory -y _/var/migration_backup_
+# {foreman-maintain} backup online \
+--preserve-directory \
+--skip-pulp-content \
+-y \
```

Comment:
This could be optional too as above?

Reply:
Yes, removed here and one more instance in another file.